pandas/core/groupby/ops.py

```diff
@@ -823,7 +823,7 @@ def groups(self) -> dict[Hashable, np.ndarray]:
     @cache_readonly
     def is_monotonic(self) -> bool:
```

Comment:
deprecate here too?

Reply:
I'll try to address this one in another pass. This appears to be a private-ish version for groupby specifically.